src/Features/VisualBasic/Portable/CodeRefactorings/UseNamedArguments/VisualBasicUseNamedArgumentsCodeRefactoringProvider.vb

```diff
@@ -47,6 +47,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.UseNamedArguments
             Protected Overrides Function IsCloseParenOrComma(token As SyntaxToken) As Boolean
                 Return token.IsKind(SyntaxKind.CloseParenToken, SyntaxKind.CommaToken)
             End Function
+
+            Protected Overrides Function SupportsNonTrailingNamedArguments(options As ParseOptions) As Boolean
+                Return False
```

Comment:
VB 15.5 does support non-trailing named arguments too. Next PR? ;-)

Reply:
oh, it does?  i didn't know you did that :)